src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -470,13 +466,23 @@ class VAMap extends Component {
     />
   );
 
+  renderSearchControls = currentQuery => (
+    <SearchControls
+      currentQuery={currentQuery}
+      onChange={this.props.updateSearchQuery}
+      onSubmit={this.handleSearch}
+      suppressCCP={this.props.suppressCCP}
+      suppressPharmacies={this.props.suppressPharmacies}
+      getRefInput={this.getRefInput}
```

Comment:
Actually we don't need `getRefInput={this.getRefInput}` you can remove it on this PR or I am already removing it in coming PR